### ISSUE TITLE
iframe support

### ISSUE THREAD

**Issue by ssshake** [michitaro/vue-window#13]
I used your package to build a system that shows all of my home networks web based services as windows. 

This is done be simply embedding iframes in the windows.

However dragging and resizing breaks when the mouse hovers over the iframe. This is impossible to avoid because as soon as you move the mouse on drag/resize it will go over the iframe before the drag or resize catches up.

All that needs to happen is for on mouse down set the children of the window to 'pointer-events:none', and on mouse up set 'pointer-events:auto'

I'm having a hell of a time doing this, I can't seem to get any mouse event to register at all for the titlebar or resize area.

![image](https://user-images.githubusercontent.com/1675868/47518841-fe017f00-d859-11e8-9185-1b6f9cf77070.png)

**Comment by ssshake** [comment]
I ended up figuring out a rather aggressive and non-vue way of doing this but it has some issues. I ended up using a series of event listeners.

```

           window.addEventListener("mousedown", (event) => {
               console.log('IFRAMES DISABLED')
               const iframes = document.getElementsByClassName('iframe');
               for (let iframe of iframes){
                   iframe.classList.add('no-events')
               }
           })
           window.addEventListener("mouseup", (event) => {
               console.log('IFRAMES ENABLED')
               const iframes = document.getElementsByClassName('iframe');
               for (let iframe of iframes){
                   iframe.classList.remove('no-events')
               }
           })  


```

So this now works for moving windows. Any time a mouse button is held down, like when moving a window, all iframes will have their pointer-events disabled so that they cannot mess with the movement. This is working.

However it seems like your code is blocking propagation of events on the resize perimeter of each window. So what I have to do is something like

```<hsc-window :resizable="true" v-on:resize="blockIframeEvents">```

But this issue is there is only an event fired upon resize, there is no event fired when resizing is done (when mouse is up). So I can't see any clean way to re-enable iframe pointer-events after resizing. I have to manually click anywhere in the window to turn it back on using a different listener I set up.

So it's like this:

1) Upon resizing of any window, disabled pointer-events for ALL iframes, because any mouse over of any iframe will swallow the event and screw up resizing
2) After resizing there is no way to detect that we are no longer resizing, so every iframe still has its pointer-events disabled
3) Listen for a click event anywhere in the document and then enable iframe pointer-events again so that iframes can be interacted with.
